cli/tests/integration/object_store.rs

```diff
@@ -9,8 +9,8 @@ async fn object_store() -> TestResult {
         authors = ["Jill Bryson <jbryson@fastly.com>", "Rose McDowall <rmcdowall@fastly.com>"]
         language = "rust"
         [local_server]
-        object_store.empty_store = []
-        object_store.store_one = [{key = "first", data = "This is some data"},{key = "second", path = "../test-fixtures/data/object-store.txt"}]
+        object_stores.empty_store = []
```

Comment:
should we also allow the previous name?

Reply:
I added backwards compatibility in 1ba490e.